apps/user_ldap/lib/LDAP.php

```diff
@@ -317,6 +317,14 @@ protected function invokeLDAPMethod(string $func, ...$arguments) {
 
 	private function preFunctionCall(string $functionName, array $args): void {
 		$this->curArgs = $args;
+		if(strcasecmp($functionName, 'ldap_bind') === 0) {
+			// The arguments are not key value pairs
+			// \OCA\User_LDAP\LDAP::bind passes 3 arguments, the 3rd being the pw
+			// Remove it via direct array access for now, although a better solution could be found mebbe?
+			// @link https://github.com/nextcloud/server/issues/38461
+			$args[2] = '***REMOVED SENSITIVE VALUE***';;
```

Comment:
```suggestion
			$args[2] = '***REMOVED SENSITIVE VALUE***';
```
One should be enough :)

Reply:
god damnit :laughing: